DuckDuckGo.xcodeproj/project.pbxproj

```diff
@@ -6052,6 +6062,24 @@
 			productReference = 98A54A8122AFCB2C00E541F4 /* Instruments.instrdst */;
 			productType = "com.apple.product-type.instruments-package";
 		};
+		B6DFE6CE2BC7E47500A9CE59 /* SwiftLintToolBundle */ = {
+			isa = PBXNativeTarget;
+			buildConfigurationList = B6DFE6D02BC7E47500A9CE59 /* Build configuration list for PBXNativeTarget "SwiftLintToolBundle" */;
+			buildPhases = (
+				B6DFE6CB2BC7E47500A9CE59 /* Sources */,
+				B6DFE6CC2BC7E47500A9CE59 /* Frameworks */,
+				B6DFE6CD2BC7E47500A9CE59 /* Resources */,
+			);
+			buildRules = (
+			);
+			dependencies = (
+				B6DFE6D62BC7E47F00A9CE59 /* PBXTargetDependency */,
+			);
+			name = SwiftLintToolBundle;
+			productName = SwiftLintToolBundle;
+			productReference = B6DFE6CF2BC7E47500A9CE59 /* .bundle */;
```

Comment:
This actually generates a ".bundle" directory (and shows a ".bundle" entry in Products on the project files list), and while it's not a big deal (everything works), let's maybe rename it properly (SwiftLintTool.bundle?) to reduce confusion.